CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [0.8.12] - 2025-06-06
+
+### Dependency updates
+
+- Bump grpcVersion to 1.73.0 (#327)
+- Bump com.nimbusds:nimbus-jose-jwt to 10.3 (#323)
+- Bump io.netty:netty-transport-native-kqueue to 4.2.1.Final (#321)
+- Bump com.google.protobuf:protobuf-gradle-plugin to 0.9.5 (#314)
+- Bump jupiterVersion to 5.13.0 (#330)
+- Bump uk.org.webcompere:system-stubs-core to 2.1.8 (#320)
```

Comment:
I believe this dependency is only used for unit testing? Also I thought we reverted this dependency version back to 2.0.3?